### PR TITLE
Bake in yq dependency in Kraken container images

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit also updates ppc64le image to have the latest bits.

### CODE REVIEW

File: containers/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # Dockerfile for kraken
 
-FROM quay.io/openshift/origin-tests:latest as origintests
```
Comment (paigerube14) [inline containers/Dockerfile:3]: I dont think you can delete this

Comment (chaitanyaenr) [reply containers/Dockerfile:3]: We used to use this image to copy the kubectl and oc clients, we are now installing them from a different source. Thoughts?

Comment (paigerube14) [reply containers/Dockerfile:3]: Ahh I see now, didnt see them at the bottom

**Comment by paigerube14** [comment]
lets let the checks run and i think we are good to merge

**Review by paigerube14 — APPROVED** [review]

**Pull request merged**